opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -297,6 +297,19 @@ enum AggregationTemporality {
   AGGREGATION_TEMPORALITY_CUMULATIVE = 2;
 }
 
+// DataPointFlags is defined as a protobuf 'uint32' type and is to be used as a
+// bit-field representing 32 distinct boolean flags.
+enum DataPointFlags {
+  FLAG_NONE = 0;
+
+  // This DataPoint is valid but has no recorded value.  This value
+  // SHOULD be used to reflect explicitly missing data in a series, as
+  // for an equivalent to the Prometheus "staleness marker".
+  FLAG_NO_RECORDED_VALUE = 1;
```

Comment:
Is this the mask or the bit position? How do we want people to use these values?

`flags & FLAG_NO_RECORDED_VALUE == 1` // The new value will be FLAG_FOO = 2, FLAG_BAR = 4
OR
`flags & (1 << FLAG_NO_RECORDED_VALUE) == 1` // The current value needs to change to 0 and we have values from 0 to 31.

Reply:
I was assuming these are masks. I've added a note.